sale_procurement_group_by_line/model/sale.py

```diff
@@ -34,10 +35,7 @@ def _compute_get_picking_ids(self):
                 [('group_id', 'in', list(group_ids))])
             sale.delivery_count = len(sale.picking_ids)
 
-    picking_ids = fields.Many2many('stock.picking',
-                                   compute='_compute_get_picking_ids',
-                                   method=True,
-                                   string='Picking associated to this sale')
+    picking_ids = fields.Many2many(compute='_compute_picking_ids')
```

Comment:
I would say that this isn't even needed as it is correctly declared in standard odoo :thinking: